### ISSUE TITLE
DATV Demodulator DVB-S FEC 7/8 broken

### ISSUE THREAD

**Issue by KR0SIV** [f4exb/sdrangel#383]
I am nolonger able to demod DATV (DVB-S) I'm using the same setup I have in the past using both a pluto and LimeSDR Mini as the transmitter to no avail.

![image](https://user-images.githubusercontent.com/13837655/61757505-e3c7e380-ad8e-11e9-9f90-6828d7f7a44c.png)

FEC 7/8 and 550ks/s

**Comment by f4exb** [comment]
Issue seems related to FEC 7/8. Other FECs work (test with GNUradio dvbs_tx example):

![dvbs_tx_qt grc](https://user-images.githubusercontent.com/6192319/61831915-a38c6200-ae6f-11e9-8e86-c4768ac30ebe.png)

Different symbol rates do not help.

**Comment by f4exb** [comment]
Oh-oooh this is what I see in the log when switching to 7/8 and since it is not prefixed I think this comes from leandvb:
`Code rate not implemented; proceeding anyway`

I have redesigned the code rate limits according more or less to ETSI standard. I bet 7/8 was not supported before. Since leandvb does not support it it should simply be be removed from the list.

**Comment by f4exb** [comment]
Actually issue is in my code because 7/8 should be supported. Here's the excerpt from leandvb dvb.h:
```c++
    // EN 300 421, section 4.4.3 Inner coding
    uint32_t pX, pY;
    switch (rate)
    {
    case FEC12:
        pX = 0x1; // 1
        pY = 0x1; // 1
        break;
    case FEC23:
    case FEC46:
        pX = 0xa; // 1010  (Handle as FEC4/6, no half-symbols)
        pY = 0xf; // 1111
        break;
    case FEC34:
        pX = 0x5; // 101
        pY = 0x6; // 110
        break;
    case FEC56:
        pX = 0x15; // 10101
        pY = 0x1a; // 11010
        break;
    case FEC78:
        pX = 0x45; // 1000101
        pY = 0x7a; // 1111010
        break;
    default:
        //fail("Code rate not implemented");
        // For testing DVB-S2 constellations.
        fprintf(stderr, "Code rate not implemented; proceeding anyway\n");
        pX = pY = 1;
    }
```

**Comment by f4exb** [comment]
7/8 missing from the string to enum converter...

**Comment by f4exb** [comment]
This is it! Fix is on dev branch.

**Comment by f4exb** [comment]
Fixed in v4.11.3
